src/risedevtool/config/src/main.rs

```diff
@@ -95,6 +96,7 @@ impl Components {
             Self::DynamicLinking => "[Build] Enable dynamic linking",
             Self::HummockTrace => "[Build] Hummock Trace",
             Self::Coredump => "[Runtime] Enable coredump",
+            Self::NoBacktrace => "[Runtime] Disable backtrace",
```

Comment:
Can call it `Enable backtrace`, and default to true.

Reply:
> Can call it `Enable backtrace`, and default to true.

This will break the compatibility as there's no way to always regenerate the default `env` file when upgrading RiseDev. 😕

Reply:
I used to consider this. But it will require running risedev configure explicitly to apply the default value, and otherwise the backtrace will be disabled without the explicit running. No sure if it's a good idea.